experimental/bits/simd_detail.h

```diff
@@ -308,6 +308,12 @@
 #define _GLIBCXX_SIMD_FIX_P2TS_ISSUE66 1
 // }}}
 
+#if __clang__
+#define _GLIBCXX_CONSTEXPR_SIMD
+#else
+#define _GLIBCXX_CONSTEXPR_SIMD constexpr
+#endif
```

Comment:
My naming convention has been to prefix everything with `_GLIBCXX_SIMD_`. I understand naming the macro "constexpr simd" is trying to explain the purpose of the macro. `SIMD_CONSTEXPR_SIMD` is just confusing again, though.
`<bits/c++config>` has `_GLIBCXX_USE_CONSTEXPR` which seems like a good fit here. I.e. name it `_GLIBCXX_SIMD_USE_CONSTEXPR`.
And define the `__clang__` branch to `const`.